node-installer/Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM busybox
-ARG TARGETPLATFORM
 COPY script/installer.sh /script/installer.sh
-COPY ./.tmp/${TARGETPLATFORM} /assets
+COPY ./.tmp /assets
```

Comment:
Why not copy assets for the target platform?

Reply:
It looks like GH actions doesn't use this Makefile to build the node-installer image. I'll remove this and create a proper fix in another PR.
https://github.com/adamreese/containerd-shim-spin/blob/4e9378e241321506bdfc674cf008c35ab87fd2da/.github/workflows/node-installer.yaml#L50-L55